topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md

```diff
@@ -118,24 +118,26 @@ having to launch jobs manually for each sample. You can start using the dataset
 >
 >    {% include snippets/create_new_history.md %}
 >
-> 2. Import the 12 `mzXML` files into a collection named `sacurine`
+> 2. Import the 12 `mzXML` files into a collection named `mzML`
 >    - Option 1: from a shared data library (ask your instructor)
 >    - Option 2: from Zenodo using the URLs given below
 >
->      [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3244991.svg)](https://doi.org/10.5281/zenodo.3244991)
+>      [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3757956.svg)](https://doi.org/10.5281/zenodo.3757956)
 >    ```
->    https://zenodo.org/record/3244991/files/HU_neg_048.mzML
->    https://zenodo.org/record/3244991/files/HU_neg_090.mzML
->    https://zenodo.org/record/3244991/files/HU_neg_123.mzML
->    https://zenodo.org/record/3244991/files/HU_neg_157.mzML
->    https://zenodo.org/record/3244991/files/HU_neg_173.mzML
->    https://zenodo.org/record/3244991/files/HU_neg_192.mzML
->    https://zenodo.org/record/3244991/files/QC1_002.mzML
->    https://zenodo.org/record/3244991/files/QC1_008.mzML
->    https://zenodo.org/record/3244991/files/QC1_014.mzML
+>    https://zenodo.org/record/3757956/files/Blanc05.mzML
+>    https://zenodo.org/record/3757956/files/Blanc10.mzML
+>    https://zenodo.org/record/3757956/files/Blanc16.mzML
+>    https://zenodo.org/record/3757956/files/HU_neg_048.mzML
+>    https://zenodo.org/record/3757956/files/HU_neg_090.mzML
+>    https://zenodo.org/record/3757956/files/HU_neg_123.mzML
+>    https://zenodo.org/record/3757956/files/HU_neg_157.mzML
+>    https://zenodo.org/record/3757956/files/HU_neg_173.mzML
+>    https://zenodo.org/record/3757956/files/HU_neg_192.mzML
+>    https://zenodo.org/record/3757956/files/QC1_002.mzML
+>    https://zenodo.org/record/3757956/files/QC1_008.mzML
+>    https://zenodo.org/record/3757956/files/QC1_014.mzML
 >    ```
 >
-> **TODO** Update with the right zenodo URLs
 >    {% include snippets/import_via_link.md collection=true format="mzml" collection_name="mzML" renaming=false %}
 >    {% include snippets/import_from_data_library.md astype="as a Collection" %}
 >
```

Comment:
it was a bit confusing to give the collection the name of the file format. I would have preferred mzml files or raw data

Reply:
This was a point I spent quite a time on actually. Originally it was named "sacurine" which is kind of the unofficial nickname of the study from which the raw data are from. However, we wanted it to be more linked to the nature of the data than to the biological question. In addition, we wanted it to be easily identified as raw data, as well as given at least a few information about the format. This is why we called it mzml to be written. So "Why not 'mzml file' then?" you may ask. There are two things that I considered here:
- the name you choose is going to appear in every output dataset name in the workflow: we want it short, with as less ponctuation as possible, and with a meaning not too confusing when reading output dataset names
- the name should not be mistaken as needing to be exactly the same as the Galaxy data type

This is why 'mzML' (with upper-case since the data type is in lower-case) seemed adequate here. 
Is this really the best choice? Of course this is no sure thing. But well, for now I think I will stick with it.
Thanks for the relevant comment anyway 😉